src/main/java/com/blamejared/crafttweaker/api/managers/IRecipeManager.java

```diff
@@ -89,10 +91,26 @@ default WrapperRecipe getRecipeByName(String name) {
     }
     
     @ZenCodeType.Method
+    @ZenCodeType.Getter("allRecipes")
     default List<WrapperRecipe> getAllRecipes() {
         return getRecipes().values().stream().map(WrapperRecipe::new).collect(Collectors.toList());
     }
     
+    /**
+     * Returns a map of all known recipes.
+     *
+     * @return A Map of recipe name to recipe of all known recipes.
+     */
+    @ZenCodeType.Method
+    @ZenCodeType.Getter("recipeMap")
+    default Map<ResourceLocation, WrapperRecipe> getRecipeMap() {
+        
+        return getRecipes().values()
```

Comment:
Why are you streaming over the values? Wouldn't it make more sense to stream over the whole entry?